ContactMechanics/FFTElasticHalfSpace.py

```diff
@@ -160,12 +160,12 @@ def fourier_slice(self):
                          zip(self.fourier_locations,
                              self.nb_fourier_grid_pts)))
 
-                def fft(self, arr):
-                    return np.fft.rfftn(arr.T).T
+                def fft(self, arr, arr_out):
```

Comment:
So we stay with the complicated API in muFFT ?

Reply:
Yes, we stick to this. Till does not want to change back.